conf/locale/eo/LC_MESSAGES/django.po

```diff
@@ -5045,6 +5104,18 @@ msgstr "Sérvér érrör. Pléäsé trý ägäïn ïn ä féw mïnütés. Ⱡ'σ
 msgid "No endpoint was provided for EdxNotes."
 msgstr "Nö éndpöïnt wäs prövïdéd för ÉdxNötés. Ⱡ'σяєм ιρѕ#"
 
+#: lms/djangoapps/instructor/tests/test_api.py
+#: lms/djangoapps/instructor/views/api.py
+msgid "This student (%s) will skip the entrance exam."
```

Comment:
@sarina I don't. It's from other commit: https://github.com/edx/edx-platform/commit/e21ff20375e5c837b10ce7ad6c1340597266486f#diff-1dcea18a471b3b764091199f6d021480R2555